src/TheThingsNetwork.cpp

```diff
@@ -612,18 +646,16 @@ void TheThingsNetwork::showStatus()
   debugPrintIndex(SHOW_RX_DELAY_2, buffer);
 }
 
-void TheThingsNetwork::configureEU868()
+// Puting this common fonction save 238 bytes of flash
+void TheThingsNetwork::configureChannelsFreq(uint32_t freq, uint8_t begin, uint8_t end, uint8_t start)
```

Comment:
- Seems to make more sense to make `end` inclusive
- Unclear what `begin` vs `start` means

Reply:
changed name to first (first channel), last (last channel) and firs_dr (first channel to change dr)